stable/sematext-agent/templates/daemonset.yaml

```diff
@@ -140,6 +146,13 @@ spec:
         - name: lib
           hostPath:
             path: /host/usr/lib
+        {{- if .Values.logagent.extraHostVolumeMounts }}
+        {{- range $_, $mount := .Values.logagent.extraHostVolumeMounts }}
+        - name: {{ $mount.name }}
+          hostPath:
+            path: {{ $mount.hostPath }}
```

Comment:
is it not missing the mountPath ?

Reply:
Mount path is not needed here, only in `volumeMounts` part.